Debug App/Podfile

```diff
@@ -8,8 +8,8 @@ target 'Debug App' do
   
   # Pods for Debug App
   pod 'PrimerSDK', :path => '../'
-  pod 'Primer3DS', '~> 2.2.0'
-  pod 'PrimerKlarnaSDK'
+  pod 'Primer3DS', "2.2.0"
+  pod 'PrimerKlarnaSDK', "1.1.0-b0"
```

Comment:
[Release 1.1.0](https://github.com/primer-io/primer-klarna-sdk-ios/releases/tag/1.1.0) has been released, so this beta version can be removed